catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -208,4 +225,48 @@ public void setFeatureService(FeatureService featureService) {
   public void setEndpointUtil(EndpointUtil util) {
     this.util = util;
   }
+
+  private Map<String, Object> getLatLonLiteral(String query) {
+    String queryStringNumbersOnly = query.replaceAll("[^\\.\\-\\d]", " ");
+    if (StringUtils.isBlank(queryStringNumbersOnly)) {
+      return null;
+    }
+    List<String> numbersAsStrings =
+        new ArrayList<>(Arrays.asList(queryStringNumbersOnly.trim().split("\\s+")));
+    if (numbersAsStrings.size() % 2 != 0) {
+      numbersAsStrings.remove(numbersAsStrings.size() - 1);
+      if (numbersAsStrings.isEmpty()) {
+        return null;
+      }
+    }
+    List<Double> numbers =
+        numbersAsStrings
+            .stream()
+            .map(number -> Double.parseDouble(number))
+            .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
If you change all of these ArrayList variable definitions to list you can use `Collectors.toList()` here.